### ISSUE TITLE
gamemode not closing after vinegar closes

### ISSUE THREAD

**Issue by ultradev-coder** [vinegarhq/vinegar#473]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

gamemode stays running in the background even if vinegar got closed

**Comment by apprehensions** [comment]
Do you mean Studio still continues to run after Vinegar closes?

**Comment by ultradev-coder** [comment]
> Do you mean Studio still continues to run after Vinegar closes?

no, i meant the gamemode process

**Comment by apprehensions** [comment]
Then this is not an issue of Vinegar. I asked because you could be using `gamemoderun` to use Studio, leading you to believe it is not Studio.

**Comment by ultradev-coder** [comment]
> Then this is not an issue of Vinegar. I asked because you could be using `gamemoderun` to use Studio, leading you to believe it is not Studio.

vinegar uses gamemode by default when running it

**Comment by apprehensions** [comment]
Yes, which relies on the Gamemode daemon to be on your system, running in the background.

**Comment by ultradev-coder** [comment]
alright, thanks for clarifying
